src/devices/hondaremote.c

```diff
@@ -0,0 +1,75 @@
+/* Honda Car Key
+ *
+ * Identifies event, but does not attempt to decrypt rolling code...
+ *
+ * Copyright (C) 2015 Tommy Vestermark
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+#include "rtl_433.h"
+#include "data.h"
+#include "util.h"
+
+static const char* command_code[] ={"boot", "unlock" , "lock",};
+
+static const char* get_command_codes(const uint8_t* bytes) {
+	return command_code[bytes[46] - 0xAA];
+	}
+
+static int hondaremote_callback(bitbuffer_t *bitbuffer) {
+	bitrow_t *bb = bitbuffer->bb;
+        uint8_t *bytes = bitbuffer->bb[0];
+	char time_str[LOCAL_TIME_BUFLEN];
+	data_t *data;
+
```

Comment:
Add an early termination check here. Test for more then 300 bits in the row or something like that.